languages/rlang/src/main/java/de/jplag/rlang/RListener.java

```diff
@@ -0,0 +1,105 @@
+package de.jplag.rlang;
+
+import static de.jplag.rlang.RTokenType.ASSIGN;
+import static de.jplag.rlang.RTokenType.ASSIGN_FUNC;
+import static de.jplag.rlang.RTokenType.ASSIGN_LIST;
+import static de.jplag.rlang.RTokenType.BEGIN_FUNCTION;
+import static de.jplag.rlang.RTokenType.BOOL;
+import static de.jplag.rlang.RTokenType.BREAK;
+import static de.jplag.rlang.RTokenType.COMPOUND_BEGIN;
+import static de.jplag.rlang.RTokenType.COMPOUND_END;
+import static de.jplag.rlang.RTokenType.END_FUNCTION;
+import static de.jplag.rlang.RTokenType.FOR_BEGIN;
+import static de.jplag.rlang.RTokenType.FOR_END;
+import static de.jplag.rlang.RTokenType.FUNCTION_CALL;
+import static de.jplag.rlang.RTokenType.HELP;
+import static de.jplag.rlang.RTokenType.IF_BEGIN;
+import static de.jplag.rlang.RTokenType.IF_END;
+import static de.jplag.rlang.RTokenType.INDEX;
+import static de.jplag.rlang.RTokenType.NEXT;
+import static de.jplag.rlang.RTokenType.NUMBER;
+import static de.jplag.rlang.RTokenType.PACKAGE;
+import static de.jplag.rlang.RTokenType.REPEAT_BEGIN;
+import static de.jplag.rlang.RTokenType.REPEAT_END;
+import static de.jplag.rlang.RTokenType.STRING;
+import static de.jplag.rlang.RTokenType.WHILE_BEGIN;
+import static de.jplag.rlang.RTokenType.WHILE_END;
+
+import de.jplag.antlr.AbstractAntlrListener;
+import de.jplag.rlang.grammar.RParser.ArrayAccessContext;
+import de.jplag.rlang.grammar.RParser.AssignmentContext;
+import de.jplag.rlang.grammar.RParser.BreakContext;
+import de.jplag.rlang.grammar.RParser.ComplexContext;
+import de.jplag.rlang.grammar.RParser.CompoundStatementContext;
+import de.jplag.rlang.grammar.RParser.FalseContext;
+import de.jplag.rlang.grammar.RParser.FloatContext;
+import de.jplag.rlang.grammar.RParser.ForContext;
+import de.jplag.rlang.grammar.RParser.FunctionCallContext;
+import de.jplag.rlang.grammar.RParser.FunctionDefinitionContext;
+import de.jplag.rlang.grammar.RParser.HelpContext;
+import de.jplag.rlang.grammar.RParser.HexContext;
+import de.jplag.rlang.grammar.RParser.IfContext;
+import de.jplag.rlang.grammar.RParser.IfElseContext;
+import de.jplag.rlang.grammar.RParser.IntContext;
+import de.jplag.rlang.grammar.RParser.ListAccessContext;
+import de.jplag.rlang.grammar.RParser.NamespaceAccessContext;
+import de.jplag.rlang.grammar.RParser.NextContext;
+import de.jplag.rlang.grammar.RParser.RepeatContext;
+import de.jplag.rlang.grammar.RParser.StringContext;
+import de.jplag.rlang.grammar.RParser.SubContext;
+import de.jplag.rlang.grammar.RParser.TrueContext;
+import de.jplag.rlang.grammar.RParser.WhileContext;
+
+/**
+ * Contains mapping for RLang to create JPlag tokens from ANTLR tokens
+ */
+public class RListener extends AbstractAntlrListener {
+    public RListener() {
+        header();
+        functions();
```

Comment:
imo these function names do not express what the functions do. It might be clear from the context of the class, but me should discuss whether we want to have more explicit names